game/scripts/npc/abilities/skeleton_king_reincarnation.txt

```diff
@@ -59,7 +59,7 @@
       "07"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "scepter_duration"                                "7"
+        "scepter_duration"                                "5" //OAA
```

Comment:
Reduce Aura radius to 600 while you're at it

Reply:
I want to add 50% dmg reduction instead.

Reply:
that's fair. Perhaps reduce their disable duration as well perhaps.